internal/commands/dash.go

```diff
@@ -191,6 +192,9 @@ func bindViper(cmd *cobra.Command) error {
 	if err := viper.BindEnv("kubeconfig", "KUBECONFIG"); err != nil {
 		return err
 	}
+	if err := viper.BindEnv("client-max-recv-msg-size", "CLIENT_MAX_RECV_MSG_SIZE"); err != nil {
```

Comment:
You don't need the extra binding here. We want the value prefixed with `OCTANT_` you only need to use BindEnv for values that don't have that prefix

Reply:
got